docs/design.md

```diff
@@ -183,18 +299,24 @@ Not implemented yet.
 
 Steps 3 and 4 can take some time, so the driver must ensure idempotency somehow.
 
-#### NodeUnstageVolume
-
-Just unmount the volume.
-
 #### NodePublishVolume
 
 Just bind-mount the volume.
 
+#### NodeUnstageVolume
+
+Just unmount the volume.
```

Comment:
I understand you didn't add it, but suggest dropping "Just" from all these sections.